src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderSettingsDaoTest.java

```diff
@@ -0,0 +1,248 @@
+package bio.terra.service.snapshotbuilder;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertThrows;
```

Comment:
When updating to JUnit 5, make sure to change these imports too.
```suggestion
import static org.hamcrest.MatcherAssert.assertThat;
import static org.junit.jupiter.api.Assertions.assertThrows;
```